dot/network/service.go

```diff
@@ -415,17 +414,15 @@ func (s *Service) sentBlockIntervalTelemetry() {
 		}
 		finalizedHash := finalised.Hash()
 
-		err = telemetry.SendMessage(telemetry.NewBlockIntervalTM(
+		s.telemetry.SendMessage(telemetry.NewBlockIntervalTM(
```

Comment:
Is it ok to ignore the error? :thinking:

Reply:
Never mind, didn't see `SendMessage` no longer returns an error

Reply:
`SendMessage` does not return error